site/reference/pkg/get/README.md

```diff
@@ -9,11 +9,63 @@ description: >
     Fetch a package from a git repo.
 -->
 
-{{< asciinema key="pkg-get" rows="10" preload="1" >}}
-
 Get fetches a remote package from a git subdirectory and writes it to a new
```

Comment:
"`get` fetches ..."

Please do an audit for this.

Reply:
I have gone through the changes and I think I have updated all mentions of the commands to use verbatim quotes.